helmfile/values/service-cluster-networkpolicy.yaml.gotmpl

```diff
@@ -92,3 +93,11 @@ dex:
 
 falco:
   enabled: {{ and .Values.falco.enabled .Values.networkPolicies.falco.enabled }}
+
+coredns:
+  enabled: {{ .Values.networkPolicies.coredns.enabled }}
+  externalDns:
+    ips: {{- toYaml .Values.networkPolicies.coredns.externalDns.ips | nindent 6 }}
+
+dnsAutoscaler:
+  enabled: {{ .Values.networkPolicies.dnsAutoscaler.enabled }}
```

Comment:
This isn't needed, right?

```suggestion
```

Reply:
It is not needed
I think they reappeared at rebase or something ...